examples/CRISP/Dockerfile

```diff
@@ -21,12 +21,14 @@ RUN apt-get update && apt-get install -y \
 RUN curl -fsSL https://deb.nodesource.com/setup_22.x | bash - \
     && apt-get install -y nodejs \
     && npm install -g corepack concurrently wait-on \
-    && corepack enable
+    && corepack enable \
+    && corepack prepare pnpm@10.7.1 --activate
 
 # Create user
 ENV USERNAME=devuser
-ENV USER_UID=1000
-ENV USER_GID=1000
+ARG USER_UID=1000
```

Comment:
is this correct as ARG and not ENV?

Reply:
these get passed in on from the local env here:

https://github.com/gnosisguild/enclave/blob/78effbf1e48a335db2a6ed53e2dd5d747f6c128d/examples/CRISP/docker-compose.yaml#L8-L9

you can override these if you don't have the default linux user id